src/server/server.ts

```diff
@@ -47,6 +47,15 @@ function isOfficeHost(host: string) {
     return officeHosts.indexOf(host) >= 0;
 }
 
+let assetPaths = null;
+function getAssetPaths() {
+    if (assetPaths === null) {
```

Comment:
>assetPaths === null [](start = 8, length = 19)

could probably just do "!assetPaths".  Since assetPaths is expected to be an object (i.e., empty string or "false" are not valid values for it in any case), I think just doing "!assetPaths" is fine.  Or check for null *or* undefined (if you leave it as undefined, per above)